### PR TITLE
Add Promo Status for New Model Inputs

### LINKED ISSUE #561: Separate recovered/deceased cases from case count for projection
**Is your feature request related to a problem? Please describe.**
As states are reaching steady state in fighting COVID, lots of incarcerated individuals have recovered and a state's cumulative case count is no longer the best indicator of their projection (because the model currently assume mid-outbreak on the cumulative cases). 

**Describe the solution you'd like**
Allow states to tell us how many of their covid cases are "recovered" or "deaths" so we can separate those from the projections.

1. (UI) In the add cases modal, add a column of inputs between "CURRENT CASES (CUMULATIVE)" and population labeled "RECOVERED CASES"
2. (UI) Add a second column to the right of recovered labeled "DEATHS"
3. (UI) Change the label of "CURRENT CASES (CUMULATIVE)" to "CUMULATIVE CASES"
4. (UI) Repeat 1-3 in the facility detail view
5. (UI) Update the copy in the add cases modal to say "Enter the number of cases detected and the total population as of this date."
6. (Model calculation) In creating the projection curve, we will now be projecting based on "active" cases only, which is cumulative - recovered - deaths.

**Describe alternatives you've considered**
- We could have states directly enter "active" cases but it is less common for states to have that data readily available.
- We could have the model take "new" cases instead of cumulative cases, but may not be a number states readily have available.
- We could not make the change and leave it as cumulative cases, but we risk some states abandoning the model because it does not accurately represent their system at present anymore.

The proposed approach also aligns with our shadow data collection efforts, which are tracking the number of recovered cases.

**Additional context**
There will be a few model updates as a part of this change:
1. Updating the projection to distribute cases across the active and recovered buckets 
2. Update the r(t) calculation so it becomes a smoothed 7-day rolling average
3. Remove the r(t) hack for discarding duplicate counts

### PR DESCRIPTION
**Pull request by neeharmb**
## Add Promo Status for New Model Inputs

> Adding recovered cases and deaths is a breaking change that will impact how projection values appear. This promo notifies the user so they are aware of this change.

Note: The changes in `src/infection-model/seir.tsx` are coming from Prettier; once that is run in `jkunz/initialize-recovered-seir` that file should no longer appear in this PR. 

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

Closes #561

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/page-multi-facility/ScenarioSidebar.tsx [hunk 1]
```diff
@@ -83,6 +85,12 @@ const promoTexts: { [promoType: string]: string } = {
     for that facility. To get an accurate Rt, enter case counts for previous
     days by clicking the number of cases on the right (in red) to update case
     numbers.`,
+  newModelInputs: `New! We’ve updated our model to factor in recoveries and 
+    deaths to enable more accurate projections based on active cases. This 
+    may have caused some of your projection values to change. You can input 
+    recoveries and deaths when you “Add Historical Data” for a facility.
```
Comment (jessex) [inline src/page-multi-facility/ScenarioSidebar.tsx:91]: Nit but the double quotes in here are not the "standard" double quotes and may render strangely in some browsers, if I recall. Usually you get these when they are copied and pasted from somewhere, but we should replace them.

File: src/page-multi-facility/CreateBaselineScenarioPage.tsx [hunk 2]
```diff
@@ -47,6 +47,7 @@ const CreateBaselineScenarioPage: React.FC = () => {
         dataSharing: true,
         dailyReports: true,
         addFacilities: true,
+        newModelInputs: false,
```
Comment (jessex) [inline src/page-multi-facility/CreateBaselineScenarioPage.tsx:50]: Not sure I immediately follow: why is this one set to false?

File: src/page-multi-facility/ScenarioSidebar.tsx (hunk 1 above)
Comment (neeharmb) [reply src/page-multi-facility/ScenarioSidebar.tsx]: good catch, thanks!

File: src/page-multi-facility/CreateBaselineScenarioPage.tsx (hunk 2 above)
Comment (neeharmb) [reply src/page-multi-facility/CreateBaselineScenarioPage.tsx:50]: Essentially, we only want this promo to be shown for existing scenarios (newly created ones should already be using recovered cases/deaths), so we set it to false here. Existing scenarios won't have this promo status yet, so we set it to true in `src/database/type-transforms.tsx` only if it is undefined, if that makes sense

**Pull request merged**